modules/core/src/TCModel.ts

```diff
@@ -140,6 +140,10 @@ export class TCModel extends Cloneable<TCModel> {
 
     }
 
+    /**
+     * created should be initialized outside of the updated method since it can have a different value.
```

Comment:
Quick question : What is the issue with `created` and `lastupdated` using both `utcdate` at the same time that needs to be solved by the initialisation of `created` outside?